help/en/html/hardware/cmri/CMRI.shtml

```diff
@@ -1362,6 +1367,7 @@
       <p style="margin-left:2em;">C/MRI bits 1, 2, 3, 4 set to 1 display as F (computer bits 0, 1,
       2, 3)<br>
       C/MRI bits 2, 4, 8 set to 1 display as 8A (computer bits 1, 3, 7)<br>
+      C/MRI bits 2, 3 set to 1 display as 10 3 (computer bits 1, 2, preceded by the escape byte value of "10")<br>
```

Comment:
Would this be C/MRI bits 1 &2 instead of 2&3?

Reply:
Fixed. Thanks.